### DISCUSSION TITLE
Welcome to CodeceptJS Discussions!

### DISCUSSION THREAD

**Discussion by DavertMik** [codeceptjs/CodeceptJS#2781] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vid** [comment]
Hi, thought I'd try this. I've been dabbling with browser and particularly BDD testing for a while, it's a very interesting way to try to bridge worlds. I work in policy, the browser is a key way people interact with the digital world, there are a lot of opportunities to understand and perhaps improve from the inside. So thanks for this great project and the good work behind it.
